<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="6" hostname="ip-172-31-23-194" name="TestCases.GuestUser" tests="7" failures="0" timestamp="24 Jul 2018 11:23:59 GMT" time="0.099" errors="1">
  <testcase name="Browser_URL_SetUP" time="0.099" classname="TestCases.GuestUser">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at globals.Globals.ERtakeScreenshot(Globals.java:180)
at TestCases.BaseTest.Browser_URL_SetUP(BaseTest.java:72)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:168)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:105)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:379)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:340)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:125)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:413)
]]>
    </error>
  </testcase> <!-- Browser_URL_SetUP -->
  <testcase name="Verify_Home_Screen_Header_Footer" time="0.000" classname="TestCases.GuestUser">
    <skipped/>
  </testcase> <!-- Verify_Home_Screen_Header_Footer -->
  <testcase name="Guest_User_SearchResult_MiniCart_VAlidations" time="0.000" classname="TestCases.GuestUser">
    <skipped/>
  </testcase> <!-- Guest_User_SearchResult_MiniCart_VAlidations -->
  <testcase name="Guest_User_BrowseCategory_MiniCart_Validations" time="0.000" classname="TestCases.GuestUser">
    <skipped/>
  </testcase> <!-- Guest_User_BrowseCategory_MiniCart_Validations -->
  <testcase name="Browse_Categories" time="0.000" classname="TestCases.GuestUser">
    <skipped/>
  </testcase> <!-- Browse_Categories -->
  <testcase name="Search_Result_Display" time="0.000" classname="TestCases.GuestUser">
    <skipped/>
  </testcase> <!-- Search_Result_Display -->
  <testcase name="Guest_User_CheckOut_Frm_MiniCart" time="0.000" classname="TestCases.GuestUser">
    <skipped/>
  </testcase> <!-- Guest_User_CheckOut_Frm_MiniCart -->
</testsuite> <!-- TestCases.GuestUser -->
